### DISCUSSION TITLE
Fuentes posibles

### DISCUSSION THREAD

**Discussion by luciano-stefano-meloclaps** [luciano-meloclaps/Gatuna_Matata#5] [Ideas]
[Best Free Fonts for Designers (unblast.com)](https://unblast.com/fonts/)
[DaFont - Descargar fuentes](https://www.dafont.com/es/)
[Browse Fonts - Google Fonts](https://fonts.google.com/)
[Adobe Fonts | Explorar fuentes ilimitadas](https://fonts.adobe.com/)

**Comment by ezeq-py** [comment]
https://www.figma.com/file/lLSNjRISBm3UtoHziEHdnP/Meninas?type=design&t=Gebn1k1WaeizddXN-6

**Reply by ezeq-py** [reply]
--text: #0e0b0e;
--background: #f3f1f4;
--primary: #ebbdd7;
--secondary: #ca96cf;
--accent: #99008c;

**Reply by ezeq-py** [reply]
https://www.figma.com/community/file/1042703573403941609/cat-illustration-pack
